### DISCUSSION TITLE
豪听

### DISCUSSION THREAD

**Discussion by Hjhnull** [share121/inter-knot#5303] [其他]
https://github.com/user-attachments/assets/241a4c6a-13df-4e3d-8971-0e17f2c6dbba

**Comment by Hjhnull** [comment]
https://mvwebfs.tx.kugou.com/202409221658/e3d3831491d15f33057d7f231854b0f1/v2/b0e151c356174bdd7c21155b4769bc18/G099/M08/1C/19/A4cBAFjnOQyAf1KOBZRArKAG1sQ795.mp4

**Comment by Jason555647** [comment]
不是😑

**Comment by Jason555647** [comment]
可恶，被骗了

**Comment by Loxic123** [comment]
装都不装了是吧<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
